com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Components/Bloom.cs

```diff
@@ -114,6 +134,10 @@ public bool highQualityFiltering
         [SerializeField, FormerlySerializedAs("resolution")]
         private BloomResolutionParameter m_Resolution = new BloomResolutionParameter(BloomResolution.Half);
 
+        [Tooltip("When enabled, bloom uses multiple bilinear samples for the prefiltering pass.")]
+        [SerializeField]
+        private BoolParameter m_HighQualityPrefiltering = new BoolParameter(true);
```

Comment:
We usually default to whatever is the medium preset, so this should be false (or true in the medium preset, but in that case we need to profile)